test/integration/graphql_test.lua

```diff
@@ -142,7 +142,7 @@ g.test_upload = function()
     )
 
     t.assert_error_msg_equals(
-        'Could not coerce "8589934592" to "Int"',
+        'Could not coerce value type "int" with value "8589934592" to type "Int"',
```

Comment:
I'm not sure it's grammatically correct. "value with value" - looks strange.

Reply:
Maybe `Could not coerce type "int" with value "8589934592" to type "Int"` will be better

Reply:
`Could not coerce value "8589934592"  (type: "int") to type "Int"`. However it looks really strange - "type int to Int". "8589934592" actually is not an int (but it seems to be parser issue).